### ISSUE TITLE
Unexpected '.' when using .call()

### ISSUE THREAD

**Issue by gibwar** [douglascrockford/JSLint#107]
``` javascript
/*jslint browser: true */

var admin = {};
(function (jQuery) {
    "use strict";
}).call(admin, window.jQuery);
```

Using the stripped sample above, the following warning is generated

> Problem at line 6 character 3: Unexpected '.'.

I am using the pattern described in [My Favorite JavaScript Design Pattern](http://www.sitepoint.com/my-favorite-javascript-design-pattern/) and [The Anatomy of a JavaScript Design Pattern](http://www.sitepoint.com/the-anatomy-of-a-javascript-design-pattern/) and use `.call(context, arg1)` to apply the encapsulated function to my single global object and pass in other global objects as parameters.

Unfortunately, this syntax is needed to evaluate the function before using `.call` and needs the wrapping parenthesis. At this time I am unsure of how to edit this design pattern that will pass validation.
